arkouda/series.py

```diff
@@ -701,21 +701,19 @@ def sort_values(self, ascending: bool = True) -> Series:
         -------
         A new Series sorted smallest to largest
         """
+        from arkouda.util import is_numeric
 
         if not ascending:
-            if isinstance(self.values, pdarray) and self.values.dtype in (
-                int64,
-                float64,
-            ):
+            if isinstance(self.values, pdarray) and is_numeric(self.values):
                 # For numeric values, negation reverses sort order
-                idx = argsort(-self.values)
+                perm = argsort(-self.values)
             else:
                 # For non-numeric values, need the descending arange because reverse slicing
                 # is not supported
-                idx = argsort(self.values)[arange(self.values.size - 1, -1, -1)]
+                perm = argsort(self.values)[arange(self.values.size - 1, -1, -1)]
```

Comment:
huh, i'm not sure when / if this comment used to be true. But it's not the case anymore

```python
>>> vals = ak.array(['c', 'a', 'b', 'c', 'c', 'b'])
>>> ak.argsort(vals)[::-1]
array([4 3 0 5 2 1])

>>> vals[ak.argsort(vals)[::-1]]
array(['c', 'c', 'c', 'b', 'b', 'a'])
```

using git blame it looks like it was present when series was originally ported over from user provided code

Reply:
Hmm, OK, I will simplify this.  Thanks for noticing and pointing this out.